src/node/extensions.ts

```diff
@@ -0,0 +1,50 @@
+import fs from "fs";
+import Module from "module";
+
+export function overrideNode() {
+  let babel: any;
+  let reactPlugin: any;
+
+  // @ts-ignore apparently _extensions doesn't exist on module? Not sure what this is doing...
+  Module._extensions[".jsx"] = (module: any, filename: string) => {
+    const content = fs.readFileSync(filename, "utf8");
+    if (!babel) {
+      babel = require("babel-core");
```

Comment:
These two babel packages are unmaintained, should probably be `@babel/core` and `@babel/plugin-transform-react-jsx`

Reply:
Makes sense to upgrade this, but but probably in another PR as it's not related to TS, Husky, etc.

Reply:
Please see https://github.com/entropic-dev/ds/pull/2#discussion_r301457809.